standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1825,6 +1827,20 @@ public enum ConfVars {
         "hive.metastore.properties.servlet.auth", "jwt",
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    ICEBERG_CATALOG_SERVLET_PATH("hive.metastore.catalog.servlet.path",
+        "hive.metastore.catalog.servlet.path", "icecli",
```

Comment:
CLI is a Command Line Interface. Is it a proper approach calling a REST service as a CLI?

Reply:
May be not.

Reply:
maybe simply `iceberg`?

https://github.com/unitycatalog/unitycatalog/blob/5f868a7e08cafbe3235887d34747e357a292fa17/server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java#L256